docs/components/collapse.md

```diff
@@ -160,8 +160,8 @@ To set the default state (collapsed or expanded) without binding to v-model you
   <template #body>
     <KCollapse title="Look Mah!">
       <template #trigger-content>
-        <div class="d-flex">
-          <KIcon icon="help" size="16" color="var(--blue-500)" class="mr-2" />
+        <div class="trigger-wrapper">
+          <KIcon icon="help" :size="KUI_ICON_SIZE_30" color="var(--blue-500, var(--kui-color-text-primary, $kui-color-text-primary))" class="icon-style" />
```

Comment:
issue: you cannot use SCSS variables in the template.

Reply:
fixed